docs/go_getting_started.md

```diff
@@ -101,10 +101,42 @@ anycable-go --port=443 -ssl_cert=path/to/ssl.cert -ssl_key=path/to/ssl.key
 => INFO time context=http Starting HTTPS server at 0.0.0.0:443
 ```
 
-## Docker
+## Deploying
+### Docker
 
 Official docker images available at [DockerHub](https://hub.docker.com/r/anycable/anycable-go/).
 
 ### Heroku
 
 See [heroku-anycable-go](https://github.com/anycable/heroku-anycable-go) buildpack.
+
+### Systemd
+
+To run on systemd you can:
```

Comment:
I think it's better to expand a bit and describe _why_ we recommend to run anycable-go as a service.

Something like this:
> We recommend running anycable-go as a system service on production servers for better manageability. On most modern Linux distributions this can be done by declaring a systemd service like this:

Reply:
Added